integrations/interface-wp-service-worker-integration.php

```diff
@@ -17,6 +19,8 @@ interface WP_Service_Worker_Integration {
 	 *
 	 * @since 0.2
 	 *
+	 * @deprecated 0.7 Integrations will be removed in the future versions.
```

Comment:
Are deprecation notices needed on each method since the class as a while has it?

Reply:
Since deprecation notice has been added to the Interface itself we don't need them on methods. But one more thing can be helpful like ~~register~~() can gain more dev attention if he is using methods from deprecated interface. 

@westonruter Need your review on this one. Please let me know if we need to remove the notice from the methods.

Reply:
@westonruter Need your suggestion here.
 In case you missed.